DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -55,7 +55,7 @@ public function process(ContainerBuilder $container)
                 $parentDefinition = null;
 
                 // NEXT_MAJOR: Remove check for DefinitionDecorator instance when dropping Symfony <3.3 support
-                if ($definition instanceof ChildDefinition || $definition instanceof DefinitionDecorator) {
+                if ($definition instanceof ChildDefinition || !class_exists(ChildDefinition::class) && $definition instanceof DefinitionDecorator) {
```

Comment:
This looks risky. Please add some brackets

Reply:
Done